solutions/ng-rx/src/app/services/todo.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Todo } from './../store/todo.model';
+import { Injectable } from '@angular/core';
+import { Http } from '@angular/http';
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/map';
+
+@Injectable()
+export class TodoService {
+  constructor(private http: Http) { }
+
+  getTodos(): Observable<Todo[]> {
+    return this.http.get('http://localhost:3000/todos')
+      .map(res => res.json());
+  }
```

Comment:
I like using the `json-server` it gives people a real http experience without relying on an external server that might not be available the day of the training. That has happened before